### PR TITLE
Start mcelog on boot and check it for memory errors

### PR DESCRIPTION
**Pull request by ulyssesbbg**

### CODE REVIEW

**Comment by j2sol** [comment]
retest

File: roles/common/defaults/main.yml [hunk 1]
```diff
@@ -35,6 +35,10 @@ common:
     tar_version: master
   system_tools:
     mcelog: True
+  service:
+    name: mcelog
+    state: restarted
+    enabled: True
```
Comment (masteinhauser) [inline roles/common/defaults/main.yml]: This doesn't actually look to be needed or even used at all?

Comment (j2sol) [inline roles/common/defaults/main.yml]: What is this change doing here?

Comment (ulyssesbbg) [inline roles/common/defaults/main.yml]: Can you please let me know the correct place to put it?  mcelog is not currently being started on boot and needs to be to catch memory errors in /var/log/mcelog

**Comment by j2sol** [comment]
Please rebase on master for Ansible 2.1 fixes.

**Comment by ulyssesbbg** [comment]
moved to https://github.com/blueboxgroup/ursula/pull/1913

**Pull request closed without merging**